tuw_nlp/common/vocabulary.py

```diff
@@ -23,7 +27,19 @@ def get_id(self, word, allow_new=True):
             raise ValueError(
                 f'{word} not in vocab and allow_new set to False')
         self.add(word)
+        self.word_to_freq[word] += 1
         return self.word_to_id[word]
 
     def get_word(self, word_id):
         return self.id_to_word[word_id]
+
+    def select_n_best(self, max_features):
+        res_largest = nlargest(
```

Comment:
Why not use `collections.Counter`, which has a `most_common` method?

Reply:
You are right @recski, It is more convenient to use, I've just commited the changes.